consoles/sshVirtsh.pm

```diff
@@ -14,6 +14,9 @@ use File::Which;
 use Mojo::DOM;
 use Mojo::File qw(path);
 use Mojo::JSON qw(decode_json);
+use Carp 'croak';
+use MIME::Base64 qw(encode_base64);
+use IO::Compress::Gzip qw(gzip :level);
```

Comment:
It would probably make sense to use `Mojo::Util:b64_encode` (see https://docs.mojolicious.org/Mojo/Util#b64_encode) and `Mojo::Util::gzip` (see https://docs.mojolicious.org/Mojo/Util#gzip). To read/write files you can use `Mojo::File`. All these functions have the advantage that error handling is less manual and the usage more streamlined. (If your code needs the detailed API these modules provide you can keep using them - but probably not.)

Reply:
Added, this works nice, thanks!